src/components/profile/profile-classic.style.js

```diff
@@ -0,0 +1,7 @@
+import { css } from 'styled-components';
+import { THEMES } from '../../style/themes';
+
+
+export default ({ theme }) => theme.name === THEMES.classic && css`
+    color: rgba(0, 0, 0, 0.85);
+`;
```

Comment:
there is no need to create a separate file if it's only one line. You could just put it on the bottom of a styled-component.

Reply:
Ye, I agree with you